sormas-ui/src/main/java/de/symeda/sormas/ui/customexport/ExportConfigurationsGrid.java

```diff
@@ -68,37 +86,74 @@ private void buildGrid(List<DataHelper.Pair<String, ExportGroupType>> availableP
 			.setCaption(I18nProperties.getPrefixCaption(ExportConfigurationDto.I18N_PREFIX, ExportConfigurationDto.NAME))
 			.setExpandRatio(1);
 
-		addComponentColumn((config) -> this.buildButtonLayout(config, availableProperties, propertyCaptionProvider)).setId(COLUMN_ACTIONS)
+		addComponentColumn((config) -> this.buildButtonLayout(config, availableProperties, propertyCaptionProvider, true))
+			.setId(COLUMN_ACTIONS)
+			.setCaption("");
+	}
+
+	private void buildGridPublicExportCustomConfiguration(
```

Comment:
This is similar to the method above, the only differences are a caption and a boolean flag.

Not sure the caption should be different and the flag should come as a constructor argument; this way you don't duplicate the this method and the constructor